src/main/java/com/mixpanel/android/viewcrawler/ImageStore.java

```diff
@@ -0,0 +1,138 @@
+package com.mixpanel.android.viewcrawler;
```

Comment:
I don't know how the image gets displayed on Android. But, what if a image is originally 100 x 100 and the view is only 50 x 50. Is Android going to shrink that image or cut it off?